extensions/ql-vscode/src/query-server/legacy/run-queries.ts

```diff
@@ -441,7 +441,6 @@ export async function compileAndRunQueryAgainstDatabaseCore(
         const error = result.message
           ? redactableError`${result.message}`
           : redactableError`Failed to run query`;
-        void extLogger.log(error.fullMessage);
```

Comment:
This was removed because it results in duplicate logs: the `showAndLogExceptionWithTelemetry` will also log it to the `extLogger`.